opencti-platform/opencti-front/src/private/components/settings/SettingsOrganizations.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import ListLines from '../../../components/list_lines/ListLines';
+import SettingsOrganizationsLines, { settingsOrganizationsLinesQuery } from './organizations/SettingsOrganizationsLines';
+import Loader, { LoaderVariant } from '../../../components/Loader';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import { SettingsOrganizationsLinesPaginationQuery, SettingsOrganizationsLinesPaginationQuery$variables } from './organizations/__generated__/SettingsOrganizationsLinesPaginationQuery.graphql';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import AccessesMenu from './AccessesMenu';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    margin: 0,
+    padding: '0 200px 50px 0',
+  },
+}));
+
+const SettingsOrganizations = () => {
+  const classes = useStyles();
+  const LOCAL_STORAGE_KEY = 'view-settings-organizations';
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<SettingsOrganizationsLinesPaginationQuery$variables>(LOCAL_STORAGE_KEY, {
+    searchTerm: '',
+    sortBy: 'name',
+    orderAsc: false,
+  });
+  const queryRef = useQueryLoading<SettingsOrganizationsLinesPaginationQuery>(settingsOrganizationsLinesQuery, paginationOptions);
+  const renderLines = () => {
+    const dataColumns = {
+      name: {
+        label: 'Name',
+        width: '30%',
+        isSortable: true,
+      },
+      x_opencti_organization_type: {
+        label: 'Type',
+        width: '20%',
+        isSortable: true,
```

Comment:
Ideally you could try to use the new render:() => () way of doing that to simplify the line component

Reply:
I may need your help for that :)
I tried to get inspired by what was done on Vocabularies but I don't understand how the hook useVocabularyCategory.tsx works.